ludwig/automl/base_config.py

```diff
@@ -121,19 +127,20 @@ def get_field_info(dataset: str):
         nonnull_values = len(dataframe[field].notnull())
         avg_words = None
         if dtype in ['str', 'string', 'object']:
-            avg_words = get_avg_words(dataframe[field])
+            avg_words = avg_num_tokens(dataframe[field])
         fields.append(
             FieldInfo(name=field, dtype=dtype,
                       distinct_values=distinct_values, nonnull_values=nonnull_values, avg_words=avg_words)
         )
     return fields, row_count
 
 
-def get_input_and_output_features(
+def get_features_config(
     fields: List[FieldInfo],
     row_count: int,
+    resources: Dict,
     target_name: str = None,
-) -> dict:
+) -> Dict:
```

Comment:
Here and in all other places we are using `typing.Dict` we should use `dict` if we don't specify the types of keys and values, and `Dict[keytype, valuetype]` otherwise. Just for consistency.

More info: https://stackoverflow.com/questions/37087457/difference-between-defining-typing-dict-and-dict